examples.py

```diff
@@ -442,4 +442,4 @@
 
 exampleDateAfterCurrentDate = '2 DEC 2025'
 
-exampleDateEqualCurrentDate = '4 OCT 2020'
+exampleDateEqualCurrentDate = '5 OCT 2020'
```

Comment:
Use `datetime.now()` and `strftime` and a new `MONTHS` array (maybe we should make a utils file with a single months array at this point) to make the current date for `exampleDateEqualCurrentDate` and tomorrow's date for the `exampleDateAfterCurrentDate`